src/discord/CommandHandler.js

```diff
@@ -50,7 +50,8 @@ class CommandHandler {
   }
 
   isOwner(member) {
-    return member.id == this.discord.app.config.discord.ownerId
+    return member.id == this.discord.app.config.discord.ownerId || member.roles.cache.find(r => this.discord.app.config.discord.ownerRoles.includes(r.id))
```

Comment:
The permissions model is if only a single user should be able to do it, check against the user's ID and if multiple users should be able to do it allocate a role which is then checked. Using multiple roles for this is pointless and if you're changing `override` to be a multi-user command then it might as well be with the general commander role, rather than a separate one.